provisioner/docker/config_openEuler-22.03.yaml

```diff
@@ -0,0 +1,25 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+docker:
+        memory_limit: "4g"
+        image: "bigtop/slaves:trunk-openEuler-22.03"
+
+        #repo: "http://repios.bigtop.apache.org/releases/3.1.0/centos/7/$basearch"
+repo: "file:///bigtop-home/output/"
+distro: openEuler
+components: [hdfs ,yarn ,mapreduce]
+enable_local_repo: true
+smoke_test_components: [hdfs, yarn, mapreduce]
```

Comment:
Thanks for working on it.
The purpose of this PR is to add OpenEuler support to Bigtop_toolchain, allowing for the building of OpenEuler Docker images for users and Bigtop developers. The `provisioner` module is used to deploy Bigtop components in the relevant Docker images.

To avoid distractions, it may be best to remove the `provisioner` modifications from this PR and focus solely on the Bigtop_toolchain module for OpenEuler Docker image building.

And please attach the related logs for successfully building the Docker images, including CentOS, Ubuntu, Rocky Linux, etc. 
Thank you very much.

Reply:
ok, i will remove the code of provisioner module